Zend/zend_gc.c

```diff
@@ -75,6 +75,8 @@
 /* one (0) is reserved */
 #define GC_ROOT_BUFFER_MAX_ENTRIES 10001
 
+#define GC_REFCOUNTED_STACK_INITIAL_SIZE 16
```

Comment:
Initial stack size can probably be larger, I'd go for at least 256 here.

Reply:
I see, I'll use 512 in this case.